aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -349,6 +349,8 @@ Tensor empty_like(
     namedinference::propagate_names(result, self.names());
   }
 
+  // never propagate Conjugate key
+  result.set_conj(false);
```

Comment:
which case actually twings this? Surely not at::empty, nor empty_strided?

Reply:
Here in `_resolve_conj` https://github.com/pytorch/pytorch/pull/54987/files#diff-611ff2ad7d44c22cda308a88cd2ee4ca8863e178aab8ce9614bba8f26b536c66R353-R355